### ISSUE TITLE
Citations

### ISSUE THREAD

**Issue by chraibi** [mkdocs/mkdocs#783]
I want to use citations as suggested by [Pandoc](http://pandoc.org/README.html#citations): `[@Ando1988]`.

Pandoc parses my  markdown file as expected, however my online documentation using mkdocs does not work. Nothing happens with the citation `[@Ando1988]`. 

Is there anyway to fix this? 
(or is it already fixed?)
```
mkdocs --version
```

says

```
mkdocs, version 0.15.0.dev
```

**Comment by waylan** [comment]
This could be handled by a Markdown [Extension][0]. You might check the list of [third party extensions][1] for any existing extensions which add this feature. If none exist, then someone would have to [build it][2]. Either way, this would be an independent third party extension and not part of MkDocs proper.

[0]: http://www.mkdocs.org/user-guide/configuration/#markdown_extensions
[1]: https://github.com/waylan/Python-Markdown/wiki/Third-Party-Extensions
[2]: https://github.com/waylan/Python-Markdown/wiki/Tutorial:-Writing-Extensions-for-Python-Markdown

**Comment by chraibi** [comment]
I found an excellent [article](http://blog.martinfenner.org/2013/06/19/citations-in-scholarly-markdown/) showing some simple possibilities. 

Thank you!

**Comment by Falkor** [comment]
Sorry but none of the links above offer a solution to cite centralised bibliographies from a bibtex file. 
Any hint for that ?

**Comment by waylan** [comment]
@Falkor if you didn't find anything in the list of known third party extensions, then we're not aware of it. You would need to create your own extension.

**Comment by Falkor** [comment]
OK, thanks for the clarification. 
I came across the following [list of additional extensions](https://github.com/twardoch/markdown-rundown/wiki/Extensions) and I tried the [mdx_bib](https://github.com/darwindarak/mdx_bib) one proposed in the list.
Just for the records, here are my (still incomplete) notes: 

Installed as followed (ideally within a virtual env).
```
pip install git+https://github.com/darwindarak/mdx_bib
```
Adapt the mkdocs configuration: 

```diff
    markdown_extensions:
+      - mdx_bib:
+             bibtex_file: biblio.bib
+             order: 'unsorted'
```
Assuming `biblio.bib` is placed within `docs/`
References seems to be parsed but still report a missing citation when labels comes from the main bibtex file. So may be this extension is just not working.
